gui/packages/desktop/src/main/index.js

```diff
@@ -275,6 +279,18 @@ const ApplicationMain = {
       return this._recoverFromBootstrapError(error);
     }
 
+    // fetch relays
+    try {
+      this._setRelays(await this._daemonRpc.getRelayLocations());
+    } catch (error) {
+      log.error(`Failed to fetch relay locations: ${error.message}`);
+
+      return this._recoverFromBootstrapError(error);
+    }
+
+    // start periodic updates
+    this._startRelaysPeriodicUpdates();
```

Comment:
FYI Now given we've safely received all of the required data to bootstrap the renderer process, we can start the periodic updates timer.